lib/core/Resolver.ts

```diff
@@ -25,21 +25,21 @@ export default class Resolver {
     console.info(`Resolving DID unique suffix '${didUniqueSuffix}'...`);
```

Comment:
Can we quickly update the revoke references in protocol.md? If you want, you can remove the descriptive sections, but keep the API sections as the new spec will not specify the API.